akka-persistence/src/main/resources/reference.conf

```diff
@@ -82,11 +78,64 @@ akka.persistence {
             }
         }
     }
-    # CircuitBreaker setting if not defined in plugin config section named circuit-breaker
-    default-circuit-breaker {
-      max-failures = 10
-      call-timeout = 10s
-      reset-timeout = 30s
+    
+    # Fallback settings for journal plugin configurations.
+    # These settings are used if they are not defined in plugin config section.
+    journal-plugin-fallback {
+
+      # Dispatcher for the plugin actor.
+      plugin-dispatcher = "akka.persistence.dispatchers.default-plugin-dispatcher"
+      
+      # Dispatcher for message replay.
+      replay-dispatcher = "akka.persistence.dispatchers.default-replay-dispatcher"
+      
+      # Set this to true if plugin actor has a constructor which expects plugin 
+      # configuration entry.
+      inject-config = false
+      
+      # Maximum size of a persistent message batch written to the journal.
+      max-message-batch-size = 200
+      
+      circuit-breaker {
+        max-failures = 10
+        call-timeout = 10s
+        reset-timeout = 30s
+      }
+      
+      # The replay filter can detect a corrupt event stream by inspecting 
+      # sequence numbers and writerUuid when replaying events. 
+      replay-filter {
+        # What the filter should do when detecting invalid events.
+        # Supported values:
+        # `repair` : discard events from old writers, warning is logged
```

Comment:
Think we could have different repair strategies? How about `repair-*`, so `repair-discard-old`